pkg/controller/const.go

```diff
@@ -1,7 +1,8 @@
 package controller
 
 const (
-	metricCAFile     = "/var/run/configmaps/service-ca-bundle/service-ca.crt"
-	metricHost       = "https://prometheus-k8s.openshift-monitoring.svc:9091"
-	alertManagerHost = "https://alertmanager-main.openshift-monitoring.svc:9094"
+	metricCAFile                       = "/var/run/configmaps/service-ca-bundle/service-ca.crt"
+	metricHost                         = "https://prometheus-k8s.openshift-monitoring.svc:9091"
+	alertManagerHost                   = "https://alertmanager-main.openshift-monitoring.svc:9094"
+	insecurePrometheusTokenEnvVariable = "INSECURE_PROMETHEUS_TOKEN"
```

Comment:
This is only for testing right?

Reply:
Exactly, for convenient local development